src/Mod/Path/PathScripts/PathDrilling.py

```diff
@@ -75,6 +75,8 @@ def initCircularHoleOperation(self, obj):
         obj.addProperty("App::PropertyEnumeration", "ReturnLevel", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "Controls how tool retracts Default=G99"))
         obj.ReturnLevel = ['G99', 'G98']  # Canned Cycle Return Level
         obj.addProperty("App::PropertyDistance", "RetractHeight", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "The height where feed starts and height during retract tool when path is finished while in a peck operation"))
+        obj.addProperty("App::PropertyEnumeration", "ExtraOffset", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "How far the drill depth is extended"))
+        obj.ExtraOffset = ['None', 'Tip Length', '2x Tip Length']  # Canned Cycle Return Level
```

Comment:
Please set the default value for "ExtraOffset" in the `opSetDefaultValues()` method later in the module.